keel-sql/src/test/kotlin/com/netflix/spinnaker/keel/sql/TestContainer.kt

```diff
@@ -0,0 +1,22 @@
+package com.netflix.spinnaker.keel.sql
+
+import com.netflix.spinnaker.kork.sql.test.SqlTestUtil.initDatabase
+import org.jooq.SQLDialect.MYSQL_5_7
+import org.testcontainers.containers.JdbcDatabaseContainer
+import org.testcontainers.containers.MySQLContainerProvider
+
+internal fun initTestDatabase() = initDatabase(
+  mySQLContainer.authenticatedJdbcUrl,
+  MYSQL_5_7
+)
+
+private val mySQLContainer = MySQLContainerProvider()
+  .newInstance("5.7.22")
+  .withDatabaseName("keel")
+  .withUsername("keel_service")
+  .withUsername("whatever")
+  .also { it.start() }
```

Comment:
I had to configure my own MySQL container rather than just using the JDBC URL form as the JOOQ generated classes encode the schema, so I need to set it (database name) consistently with the real database.